src/components/welcome/MyAccounts/index.tsx

```diff
@@ -148,61 +170,84 @@ const AccountsList = ({ safes, onLinkClick, isSidebar = false }: AccountsListPro
           {isSidebar && <Divider />}
 
           <Paper className={css.safeList}>
-            {/* Pinned Accounts */}
-            <Box mb={2} minHeight="170px">
-              <div className={css.listHeader}>
-                <SvgIcon
-                  component={BookmarkIcon}
-                  inheritViewBox
-                  fontSize="small"
-                  sx={{ mt: '2px', mr: 1, strokeWidth: 2 }}
-                />
-                <Typography variant="h5" fontWeight={700} mb={2}>
-                  Pinned
+            {/* Search results */}
+            {searchQuery ? (
+              <>
+                <Typography variant="h5" fontWeight="normal" mb={1} color="primary.light">
+                  Found {filteredSafes.length} result{filteredSafes.length === 1 ? '' : 's'}
```

Comment:
Not a big issue but the list is slightly "jumping" when switching between the searchQuery view and the default view because the spacing of "Found x results" and "Pinned" is a bit different.